### PR TITLE
add docs for SDK.

### LINKED ISSUE #9: Add SDK documentation and examples
There is no info about how to use the existing SDK for Python and Golang.

### PR DESCRIPTION
**Pull request by guumaster**
closes #9

### CODE REVIEW

File: content/en/docs/KRT/sdk/index.md [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+    ERR_NEW_LABELS = "new_labels"
+```
+
+
+#### 1. Saving real value to an existing prediction
+
+ 
+{{<highlight python "linenos=table,hl_lines=2-5">}}
+async def handler(ctx, input):
+	ctx.prediction.save(date="2020-04-06T09:02:09.277853Z",
```
Comment (angelsanchez) [inline content/en/docs/KRT/sdk/index.md:407]:
```suggestion
	ctx.prediction.save(utcdate=datetime.utcnow(), 
```

File: content/en/docs/KRT/sdk/index.md [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+
+
+### Metrics
+
+KRE has built-in graphs for classification problems. To populate the "Predictions" page of a Version, you need to save prediction values along with real values. 
+
+Metrics context also provides an easy way to register common errors during a classification prediction: 
+
+```python
+    ctx.metrics.
```
Comment (angelsanchez) [inline content/en/docs/KRT/sdk/index.md:394]:
```suggestion
    ctx.prediction.
```

File: content/en/docs/KRT/sdk/index.md [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+        save(predicted_value="", true_value="", date="", error="")
```
Comment (angelsanchez) [inline content/en/docs/KRT/sdk/index.md:395]:
```suggestion
        save(predicted_value="", true_value="", utcdate=None, error="") 
```

File: content/en/docs/KRT/sdk/index.md [hunk 4]
```diff
@@ -0,0 +1,426 @@
...
+    // Saves a value in the context internal storage
+	ctx.Set("greeting", "Hello")   
+}
+
+func handler(ctx *kre.HandlerContext, data []byte) (interface{}, error) {
+    // Recover value from the internal storage
+    prefix := ctx.GetString("greeting")  
+
+    // Here you can do any transformation to your data.
+	greetingText := fmt.Sprintf("%s %s!", prefix, input.Name)
```
Comment (angelsanchez) [inline content/en/docs/KRT/sdk/index.md:63]: review indent

**Comment by alejandrogr** [comment]
closing as we are re-writing all the docs.

**Pull request closed without merging**